include/grl/kuka/KukaJAVAdriver.hpp

```diff
@@ -288,9 +289,27 @@ namespace grl { namespace robot { namespace arm {
 
           auto name = fbbP->CreateString(std::get<RobotName>(params_));
 
-          auto kukaiiwaArmConfiguration = flatbuffer::CreateKUKAiiwaArmConfiguration(*fbbP,name,commandInterface_,monitorInterface_);
+          auto clientCommandMode = grl::flatbuffer::EClientCommandMode_POSITION;
```

Comment:
should this command mode be hardcoded this way? is it a default? I might just need to look at this more closely in context.